md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessAPI.java

```diff
@@ -472,27 +518,26 @@ RestWrapper zippedexport(HttpServletResponse resp,
             FileInputStream in = new FileInputStream(myFile);
             byte[] buffer = new byte[4096];
             int length;
-            while ((length = in.read(buffer)) > 0){
+            while ((length = in.read(buffer)) > 0) {
                 out.write(buffer, 0, length);
             }
             in.close();
-                out.flush();
+            out.flush();
             restWrapper = new RestWrapper(processExport, RestWrapper.OK);
-            }
-        catch (MetadataException e) {
+        } catch (MetadataException e) {
             LOGGER.error(e);
             restWrapper = new RestWrapper(e.getMessage(), RestWrapper.ERROR);
-        } catch (IOException io){
+        } catch (IOException io) {
             LOGGER.error(io);
             restWrapper = new RestWrapper(io.getMessage(), RestWrapper.ERROR);
         }
         return restWrapper;
     }
 
     @RequestMapping(value = {"/import", "/import/"}, method = RequestMethod.POST)
-    @ResponseBody public
-    RestWrapper importData(@ModelAttribute("fileString")
-                           @Valid String uploadedFileName, BindingResult bindingResult) {
+    @ResponseBody
+    public RestWrapper importData(@ModelAttribute("fileString")
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "importData" is 56 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)